samples/java-protobuf-eventsourced-customer-registry-subscriber/docker-compose.yml

```diff
@@ -1,15 +1,16 @@
 version: "3"
 services:
   # note the ports being different from other sample docker-compose files to allow this service to run
-  # on the same local machine as the scala-customer-registry-
+  # on the same local machine as the java-protobuf-eventsourced-customer-registry
   kalix-proxy:
     image: gcr.io/kalix-public/kalix-proxy:1.1.8
-    command: -Dconfig.resource=dev-mode.conf -Dlogback.configurationFile=logback-dev-mode.xml -Dkalix.proxy.http-port=9001 -Dkalix.proxy.eventing.support=google-pubsub-emulator
     ports:
-      - "9001:9001"
+      - "9001:9000"
```

Comment:
We don't need to change the port inside the container, only the exposed port.

On property less to configure.